index.html

```diff
@@ -94,19 +104,25 @@ <h1 class="tit-projects">Projects</h1>
          </div>            
       </article>
    </section>
+
+   <section class="sec-works-pc">
+      <img class="img-works-pc" src="resources/Group-117.png" alt="works-projects">
+   </section>
+
    
-   <section class="about-me">
-      <h1 class="title" id="ab-tit">About</h1>
-      <p class="parrafo"> 
-         Lorem ipsum dolor sit amet, consectetur adipiscing elit, 
-         sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
-         Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris
-         nisi ut aliquip ea eiusmod
-      </p>
-      <button class="button-secc">Get my CV -&gt; </button>
-   </section>   
+   <section id="abilities-and-about-me-flex">
+      <section id="about-me">
+         <h1 class="title" id="ab-tit">About</h1>
+         <p id="ab-descr" class="parrafo"> 
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit, 
+            sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+            Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris
+            nisi ut aliquip ea eiusmod
+         </p>
+         <button id="ab-btn" class="button-secc">Get my CV -&gt; </button>
```

Comment:
- [ ] Please make sure your hover and pressed state of all buttons match the Figma template. Kindly add a corresponding hover and pressed state, please.
## Design template
![p3](https://user-images.githubusercontent.com/94551484/186993209-02650b37-276a-423c-9d5f-d69be224d31f.png)